flash/image/instance_segmentation/cli.py

```diff
@@ -39,7 +39,7 @@ def from_pets(
     batch_size: int = 1,
     **data_module_kwargs,
 ) -> InstanceSegmentationData:
-    """Downloads and loads the pets data set from icedata."""
+    """Downloads and loads the pets' data set from icedata."""
```

Comment:
I think it should be `pet` here. (https://airctic.github.io/icedata/pets/)

```suggestion
    """Downloads and loads the pet data set from icedata."""
```